Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -170,4 +171,19 @@ private record LoadRequest(TextDocumentItem TextDocument, CancellationToken Canc
 
     private record VerifyRequest(DafnyDocument OriginalDocument, CancellationToken CancellationToken) : Request(CancellationToken);
   }
+
+  public class VerificationProgressReporter : IVerificationProgressReporter {
+    public ICompilationStatusNotificationPublisher publisher { get; init; }
+    public TextDocumentItem document { get; init; }
+
+    public VerificationProgressReporter(TextDocumentItem document,
+                                        ICompilationStatusNotificationPublisher publisher) {
+      this.document = document;
+      this.publisher = publisher;
+    }
+
+    public void reportProgress(string message) {
```

Comment:
The usual coding convention is that methods start with an upper case letter in C#.